### PR TITLE
6 Grouped Bar Chart

### PR DESCRIPTION
**Pull request by chen-chen-stat**
Added data and codes for grouped bar chart (Figure 4 in the manuscript), updated README.Rmd

### CODE REVIEW

File: README.Rmd
Comment (Lovemore-Gakava) [inline README.Rmd:1]: For the pyramid_plot chunk suggest replace with:
```suggestion
demography |>
  dplyr::mutate(
    Type = as.factor(paste0("Type ", Type)),
    figprev = ifelse(
      Gender == "Females", -1 * Prevalence / 100000, Prevalence / 100000
    ),
    Sex = Gender
  ) |>
  pyramid_chart(
    levelvar = "Type", xvar = "figprev", yvar = "Age",
    groupvar = "Sex", alpha_set = 0.7, chartcolors = colfun()$fig2_colors,
    xlab = "Prevalence (x 100 000)"
  )
```

File: README.Rmd [hunk 1]
```diff
@@ -41,3 +41,13 @@ charts::demography |>
   ) |>
   charts::pyramid_chart(Typev = c("A", "B"), gendervar = "Sex", alpha_set = 0.7)
 ```
+
+## Figure 4 - Comorbidities (Grouped Bar Chart)
+
+```{r grouped_barchart, echo = FALSE}
+charts::grouped_barchart(
```
Comment (Lovemore-Gakava) [inline README.Rmd:48]: remove reference to chart, e.g. charts::grouped_barchar to grouped_barchar and charts::comorbidities to comorbidities . In the first chunk add the line: devtools::load_all()

**Review by Lovemore-Gakava — COMMENTED** [review]
Great work @chen-chen-stat - I have added a few comments regarding the readme file. 

- Change the call to the pyramid_chart - which I should have done in the previous PR and

- a suggestion to add devtools::load_all() in the first chunk to remove the necessity to use the package reference - chart::

File: README.Rmd (hunk 1 above)
Comment (chen-chen-stat) [reply README.Rmd]: Updated as suggested

File: README.Rmd
Comment (chen-chen-stat) [reply README.Rmd:1]: Updated as suggested

**Review by Lovemore-Gakava — APPROVED** [review]

**Pull request merged**